.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
For reviewer: added this file as upgrading the pytest-nhd-apim package didn't work with Python 3.11.